frontend/src/components/BowtieInfo/BowtieInfoSection.tsx

```diff
@@ -0,0 +1,50 @@
+const BowtieInfoSection = () => {
+  return (
+    <div className="card mx-auto mb-3 w-75" id="bowtie-info">
+      <div className="card-header">About Bowtie</div>
+      <div className="card-body">
+        <table className="table table-sm table-hover">
+          <thead>
+            <tr>
+              <td>
+                Bowtie is tool for understanding and comparing implementations
+                of the &nbsp;
+                <a
+                  href={"https://json-schema.org/"}
+                  target={"_blank"}
+                  rel={"noopener noreferrer"}
```

Comment:
It's fine to include `referrer` here, linking to the JSON Schema site is essentially "internal" inasmuch as we probably would be interested to know if lots of traffic was being generated from Bowtie's site.

```suggestion
                  rel={"noopener"}
```